SCClassLibrary/Common/Math/Number.sc

```diff
@@ -61,4 +61,326 @@ Number : Magnitude {
 		^this.forBy(last, step, function)
 	}
 
+
+	/*  Boost Special Functions  */
+
+	/* Number Series */
+
+	bernouliB2n { _BernouliB2n; ^this.primitiveFailed }
+	tangentT2n { _TangentT2n; ^this.primitiveFailed }
+
+
+	/*  Gamma  */
+
+	// "true gamma" of input value
+	tgamma { _TGamma; ^this.primitiveFailed }
+	// gamma(value + 1) - 1
+	tgamma1pm1 { _TGamma1pm1; ^this.primitiveFailed }
+	// log gamma
+	lgamma { _LGamma; ^this.primitiveFailed }
+	// digamma, trigamma, polygamma
+	digamma { _Digamma; ^this.primitiveFailed }
+	trigamma { _Trigamma; ^this.primitiveFailed }
+	polygamma { |z| _Polygamma; ^this.primitiveFailed }
+	// gamma(this) / gamma(b)
+	tgammaRatio { |b| _TGammaRatio; ^this.primitiveFailed }
+	// gamma(this) / gamma(this + delta)
+	tgammaDeltaRatio { |delta| _TGammaDeltaRatio; ^this.primitiveFailed }
+	// normalised lower/upper incomplete gamma function of a and z
+	gammaP { |z| _GammaP; ^this.primitiveFailed }
+	gammaQ { |z| _GammaQ; ^this.primitiveFailed }
+	// full (non-normalised) lower/upper incomplete gamma function of a and z
+	tgammaLower { |z| _TGammaLower; ^this.primitiveFailed }
+	tgammaUpper { |z| _TGammaI; ^this.primitiveFailed }
+
+	// Incomplete Gamma Function Inverses
+	// Returns a value x such that: p = gammaP(a, x);
+	// Requires: a > 0 and 1 >= p >= 0.
+	gammaPInv { |p| _GammaPInv; ^this.primitiveFailed }
+	// Returns a value x such that: q = gammaQ(a, x);
+	// Requires: a > 0 and 1 >= q >= 0.
+	gammaQInv { |q| _GammaQInv; ^this.primitiveFailed }
+	// Returns a value a such that: p = gammaP(a, x);
+	// Requires: x > 0 and 1 >= p >= 0.
+	gammaPInvA { |p| _GammaPInvA; ^this.primitiveFailed }
+	// Returns a value a such that: q = gammaQ(a, x);
+	// Requires: x > 0 and 1 >= q >= 0.
+	gammaQInvA { |q| _GammaQInvA; ^this.primitiveFailed }
+	// Derivatives of the Incomplete Gamma Function
+	gammaPDerivative { |x| _GammaPDerivative; ^this.primitiveFailed }
+	gammaQDerivative { |x| ^this.gammaPDerivative(x).neg }
+
+
+	/*	Factorials and Binomial Coefficients */
+
+	// TODO: replace Integer:factorial?
```

Comment:
yes, if your primitive treats both  `Float` and `Integer` (in sclang) you just remove the current implementation from `Integer.sc`.

Reply:
I've removed Integer:factorial and cleared it from the Integer help doc.